com.onlyoffice.DesktopEditors.json

```diff
@@ -0,0 +1,93 @@
+{
+  "app-id": "com.onlyoffice.DesktopEditors",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "/app/desktopeditors/onlyoffice-desktopeditors.sh",
+  "separate-locales": false,
+  "tags": [
+    "AGPL-3.0"
+  ],
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--device=all",
+        "--filesystem=xdg-videos:ro",
+        "--filesystem=xdg-pictures:ro",
+        "--filesystem=xdg-download",
+        "--filesystem=host",
```

Comment:
Doesn't `filesystem=host` kinda defeat the point of the others? (BTW minor nit: the indentation here is off.)